index.html

```diff
@@ -297,11 +297,43 @@ <h2>
           For a given <a>document</a>, the <a>current orientation type</a> and
           the <a>current orientation angle</a> are strongly linked in the sense
           that for any given type, there will be a specific angle associated.
-          However, the <a>user agent</a> can associate <code>*-primary</code>
-          and <code>*-secondary</code> values at will. For example, if
-          <code>90</code> is associated with <code>landscape-primary</code> and
-          <code>270</code> with <code>landscape-secondary</code> for one
-          <a>document</a> , another one MAY get the opposite relationship.
+          However, the <a>user agent</a> can associate certain
+          <code>*-primary</code> and <code>*-secondary</code> values at will.
+          One primary orientation will always be determined by the natural
+          orientation of the device and this will then determine the secondary
+          value of its related orientation. For example a phone held in its
```

Comment:
Breaking this up at the example might be good. This paragraph is getting rather long, and the example is non-normative. 

```suggestion
          value of its related orientation. </p> <p>For example, a phone held in its
```